docs/designs/supporting_pytorch.md

```diff
@@ -0,0 +1,297 @@
+# Design doc about Supporting PyTorch
+
+## introduction
+ElasticDL is an open-source distributed deep learning programming framework based on TensorFlow 2.x and Kubernetes. At the Google Developer Day event in the fall of 2019, the ElasticDL team from Ant Financial showed off the first open-source version of ElasticDL.The primary design intent of ElasticDL is to simplify distributed programming. It allows users to provide only models described with TensorFlow 2.0 API, without requiring users to write distributed training process code. As long as the user's model definition can be adjusted locally, the model can be trained with large-scale data in a distributed environment, thereby improving R&D efficiency.
+
+However, this project only supports tensorflow. Considering that PyTorch is more widely used in academia, this project will improve and support PyTorch.
```

Comment:
tensorflow -> TensorFlow
"this project will improve and support PyTorch" -> "this project will support PyTorch"。We will not improve Pytorch.

Reply:
It is better to summary what we should do to support PyTorch here.

Reply:
The documentation needs to clearly state what is different from supporting TensorFlow frameworks. I will add this section to the document.